src/system/SystemConfig.h

```diff
@@ -764,6 +764,9 @@ struct LwIPEvent;
  *
  *  Defaults to enabled on Zephyr platforms that enable CONFIG_EVENTFD.
  */
+#ifndef CONFIG_EVENTFD
+#define CONFIG_EVENTFD 0
```

Comment:
Can we check for `defined(CONFIG_EVENTFD)`, instead? All Kconfig booleans are either defined or not, so in this case setting default values is not necessary.